packages/components/src/components/infinite-data-list/infinite-data-list.tsx

```diff
@@ -4,28 +4,30 @@ import DataList, { TRow, TRowRenderer } from '../data-list/data-list';
 
 const InfiniteLoader = _InfiniteLoader as unknown as React.FC<InfiniteLoaderProps>;
 type TInfiniteDatalist = {
-    className: string;
+    className?: string;
     data_list_className: string;
     has_more_items_to_load: boolean;
     items: TRow[];
     keyMapperFn?: (row: TRow) => number | string;
     loadMoreRowsFn: <T>(params: IndexRange) => Promise<T>;
-    onScroll: () => void;
+    onScroll?: () => void;
     rowRenderer: TRowRenderer;
-    has_filler: boolean;
-    overscanRowCount: number;
+    has_filler?: boolean;
+    overscanRowCount?: number;
     getRowSize?: (params: { index: number }) => number;
 };
 
 const InfiniteDataList = ({
     className,
     data_list_className,
-    has_filler, // Can be used as a top offset.
+    has_filler = false, // Can be used as a top offset.
     has_more_items_to_load,
     items,
     keyMapperFn,
     loadMoreRowsFn,
-    onScroll,
+    onScroll = () => {
```

Comment:
Do we need to add a default fun for this? :thinking: 
You can just use ternary operator :thinking: 

```suggestion
    onScroll,
```

Reply:
updated. removed the default function since it was handled inside datalist